capa/features/extractors/ghidra/insn.py

```diff
@@ -115,25 +113,43 @@ def extract_insn_number_features(fh: FunctionHandle, bb: BBHandle, ih: InsnHandl
         return
 
     for i in range(insn.getNumOperands()):
-        if insn.getOperandType(i) != OperandType.SCALAR:
+        # Exceptions for LEA insn:
+        # invalid operand encoding, considered numbers instead of offsets
+        # see: mimikatz.exe_:0x4018C0
+        if insn.getOperandType(i) == OperandType.DYNAMIC and insn.getMnemonicString().startswith("LEA"):
+            op_objs = insn.getOpObjects(i)
+            if isinstance(op_objs[-1], ghidra.program.model.scalar.Scalar):
+                # Additional check, avoid yielding "wide" values (ex. mimikatz.exe:0x471EE6 LEA EBX, [ECX + EAX*0x4])
+                if isinstance(op_objs[-2], ghidra.program.model.lang.Register):
+                    # EAX*0x4, note: Ghidra does not currently support this level of offset & number differentiation
+                    # all byte addressing is referred to as a DYNAMIC OperandType
+                    if op_objs[-2].getName() + "*" + hex(op_objs[-1].getValue()) in insn.toString():
+                        continue
```

Comment:
If I'm following, these checks avoid extracting `4` from instructions like `LEA EBX, [ECX + EAX*0x4]`? Can we simply check if the  instruction is `LEA` and `len(insn.getOpObjects[1])` != 4? If not, is it guarenteed that `insn.toString()` uses hex representation for numbers? e.g. if a user sets the disassembly view to base 10 representation.